esec_teardown/server.py

```diff
@@ -0,0 +1,70 @@
+from suds.client import Client
+import ssl
+import sys
+import argparse
+import logging
+import os
+
+
+LOGGER = logging.getLogger(__name__)
+
+_create_unverified_https_context = ssl._create_unverified_context
+
+logging.basicConfig(filename="./esec-delete.log", level=logging.DEBUG)
+
+url = os.getenv('ESEC_URL')
```

Comment:
3 wasted lines here as it seems your assigning 3 variables that get used only once. might as well initialise the Client with the os.getenv directly.